src/CoreBundle/Util/FieldTranslationUtil.php

```diff
@@ -219,9 +219,6 @@ private function changeTranslatableState($fieldId, $translatable)
         if (false === $fieldConf->sqlData) {
             throw new InvalidArgumentException("Field with ID '$fieldId' not found.");
         }
-        if ($fieldConf->fieldIsTranslatable === $translatable) {
```

Comment:
That looks rather much for a simple bug?

Why wasn't this a problem until now?

Reply:
The operations themselves are idempotent - if a field already has the multilanguage flag or the additional language fields, they will be unchanged.

So this change only removes a performance optimization, which isn't that much of a problem during migrations. 

This code changed in 6.2.0, so it seems that simply no updates that rely on the old behavior were executed.